app/views/recipes/new.html.erb

```diff
@@ -0,0 +1,27 @@
+<h1  align="center">Add new Recipes</h1>
+<div class="col-lg-12" id="signup">
+  <%= form_with model: @recipe, url: user_recipes_path(@user) do |form| %>
+    <div class="mb-3">
+    <%= form.text_field :name, placeholder: "Enter Your Name", class: "form-control" %>
+    </div>
```

Comment:
- [ ] Same here, kindly change the placeholder from your name to something related to Recipe name 😉